cartography/intel/gcp/cloudrun.py

```diff
@@ -140,6 +143,15 @@ def get_cloudrun_configurations(cloudrun: Resource, project_id: str, common_job_
         else:
             raise
 
+@timeit
+def transform_configurations(configurations: List[Dict], project_id: str) -> List[Dict]:
+    configurations = []
+    for item in configurations:
+        item['consolelink'] = gcp_console_link.get_console_link(project_id=project_id, resource_name='cloudrun')
```

Comment:
resource_name is invalid.

Reply:
I have added the the consolelink implementeation for new resources and raised a PR